### PR TITLE
Accepted institution email

### PR DESCRIPTION
**Pull request by raonismaneoto**
**Feature/Bug description:**
It wasn't sending any specific html to notify the user that his institution has been accepted
**Solution:**
I've created the html and connected it with the application.

screenshot from 2018-05-11 11-15-41

screenshot from 2018-05-11 11-08-20

screenshot from 2018-05-11 11-08-04

**TODO/FIXME:** n/a

### CODE REVIEW

File: backend/admin.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def get(self):
         admin.name = 'Administrador da Plataforma Virtual CIS'
         admin.cpf = '000.000.000-01'
         admin.email = [
-            'deciis@saude.gov.br',
+            'testeeciis@gmail.com',
```
Comment (mayzabeel) [inline backend/admin.py]: undo

Comment (Luiz-FS) [inline backend/admin.py]: Return to old email

File: backend/admin.py [hunk 2]
```diff
@@ -309,9 +309,9 @@ def get(self):
             'actuation_area': 'GOVERNMENT_AGENCIES',
             'description': 'Departamento do Complexo Industrial e Inovação em Saúde',
             'photo_url': 'http://www.plataformacis.org/images/logo.png',
-            'email': 'deciis@saude.gov.br',
+            'email': 'testeeciis@gmail.com',
```
Comment (Luiz-FS) [inline backend/admin.py]: Some

File: backend/models/request_institution.py [hunk 3]
```diff
@@ -26,25 +27,15 @@ def create(data):
...
-        Realize seu login no link abaixo:
-        http://frontend.plataformacis.org/signin
-
-        Equipe da Plataforma CIS""" % institution_name
-
-        sender_email = self.sender_key.get().email[0]
-        body = acceptMessage if operation == "ACCEPT" else rejectMessage
-        super(RequestInstitution, self).send_email(host, sender_email, body)
+        subject = """Instituição aceita na Plataforma Virtual CIS."""
+        email_sender = AcceptedInstitutionEmailSender(**{
```
Comment (mayzabeel) [inline backend/models/request_institution.py]: check the rejectMessage

**Pull request merged**